From: Maximiliano Curia <maxy@gnuservers.com.ar>
Date: Fri, 26 Jan 2018 13:54:30 -0300
Subject: Ignore the svg files in .pc

---
 validate_svg.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/validate_svg.sh b/validate_svg.sh
index f076a5b..2876d1e 100644
--- a/validate_svg.sh
+++ b/validate_svg.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-find . -name '*.svg' -exec xmllint --noout {} + 2> xmlerrors
+find . -path ./.pc -prune -or -name '*.svg' -exec xmllint --noout {} + 2> xmlerrors
 if [ -s xmlerrors ]; then
     cat xmlerrors
     rm xmlerrors
